packages/nimble-components/src/spinner/specs/README.md

```diff
@@ -0,0 +1,125 @@
+  
+### Features
+
+- Show indeterminate progress with an animation
+  - Cannot be paused - assumption is that component will be added to/ removed from the page as needed
+- Supports 3 sizes: `small` (16x16), `medium` (32x32) (default), `large` (64x64)
+- For the color theme only, supports an alternate appearance/color scheme via `appearance-variant="alternate"`
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+(TODO) *Screenshots and/or links to existing, canonical, or exemplary implementations of the component.*
+
+## Design
+
+The `nimble-spinner` is an animating indicator that can be placed in a particular region of a page to represent loading progress, or an ongoing operation, of an indeterminate / unknown duration.
+
+### API
+
+Component Name: `nimble-spinner`
+
+Properties/Attributes:
+- `size`
+  - String attribute controlling size of the component
+  - Backed by an enum `SpinnerSize` with values `small` (16x16), `medium` (32x32), `large` (64x64)
+  - If omitted, the default is `medium` (32x32)
+- `appearance-variant`
+    - default (undefined): Standard appearance (light/dark/color themes)
+    - `alternate`: Alternate appearance/color, for the color theme only
+    - Backed by enum `SpinnerAppearanceVariant`
+
+Methods, Events, CSS Classes, CSS Custom Properties: None
+
+### Anatomy 
+
+Contains 2 `div` elements that are animated via CSS animations.
+
+Slot Names, Host Classes, Slotted Content/ Slotted Classes, CSS Parts: None
+
+### Angular integration 
+
+Directive `NimbleSpinnerDirective` targeting selector `nimble-spinner`, with bindings for `size` and `appearance-variant`.
+
+### Blazor integration 
+
+Standard Blazor implementation (`NimbleSpinner` deriving from `ComponentBase`), with bindings for `size` and `appearance-variant`.
+
+### Visual Appearance
+
+[Visual Design spec - Adobe XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/dece308f-79e7-48ec-ab41-011f3376b49b/)
+
+Light / dark themes only have a single appearance variant.  
+The color theme has 2 appearance variants - the "Color UI Alt." one from XD can be accessed with `appearance-variant="alternate"`.
+
+## Implementation
+
+Component implementation will derive from the FAST `FoundationElement`.
+
+**Alternatives Considered:** We could derive from `fast-progress-ring` (radial progress indicator, which supports an indeterminate progress display), however then our component API would also pick up the `min`/`max`/`value` properties for determinate progress (and `paused` to pause the indicator), which we don't plan on supporting at this time. Additionally, the visual appearance of our spinner will be a little different from a radial progress bar, and we still may end up supporting a standard radial progress bar in the future.  
```

Comment:
Can you comment whether there anything we can learn/leverage/copy from the FAST progress ring implementation? For example, does the ARIA configuration you describe below match what FAST is doing?

Reply:
For this basic spinner we don't really need to leverage anything from the FAST progress ring. My ARIA plan matches what they have for an indeterminate progress ring though.

Reply:
Added that ARIA info to the spec.